VideoAPI/VideoApi.Contract/Requests/EndConsultationRequest.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace VideoApi.Contract.Requests
+{
+    public class EndConsultationRequest
```

Comment:
I can't seem to find the validator for this contract

Reply:
Changed to use LeaveConsultationRequest as room id no longer required.